CHANGES.rst

```diff
@@ -58,6 +58,9 @@ datamodels
 
 - Correcting the default ZEROFRAME allocation. [#6791]
 
+- Add the new MIRI IFU point source correction reference file data model
+  of type ifuptcor. [#6762]
```

Comment:
Perhaps better to mention the actual data model class name here, rather than the module name, e.g. `MiriIFUPtCorrModel`.